### PR TITLE
Add maiac descriptions

### LINKED ISSUE #3270: Add MAIAC descriptions
Add MAIAC descriptions

### PR DESCRIPTION
**Pull request by minniewong**
## Description

Fixes #3270 

Add MAIAC Layer Descriptions.

## Further comments

If this is a relatively large or complex change, start a discussion by explaining why you chose the solution you did and what alternatives you considered, etc...

## Checklist

This is simply a reminder of what we are going to look for before merging your code.

- I have read the [CONTRIBUTING](https://github.com/nasa-gibs/worldview/blob/master/.github/CONTRIBUTING.md) doc
- I have added necessary documentation (if applicable)
- I have added tests that prove my fix is effective or that my feature works (if applicable)
- Any dependent changes have been merged and published in downstream modules (if applicable)

@nasa-gibs/worldview

### CODE REVIEW

File: config/default/common/config/metadata/layers/modis/combined/Aerosol_Combined.md [hunk 1]
```diff
@@ -12,4 +12,10 @@ References: [NASA Earthdata - NRT Value-Added MODIS AOD Product](https://earthda
 ### MAIAC Aerosol Optical Depth
 Temporal coverage: 28 September 2019 - present
 
+The MAIAC Aerosol Optical Depth layer is from the the Moderate Resolution Imaging Spectroradiometer (MODIS) combined Terra and Aqua Multi-Angle Implementation of Atmospheric Correction (MAIAC) Land Aerosol Optical Depth level 2 product.
```
Comment (edplato) [inline config/default/common/config/metadata/layers/modis/combined/Aerosol_Combined.md:15]:
```suggestion
The MAIAC Aerosol Optical Depth layer is from the Moderate Resolution Imaging Spectroradiometer (MODIS) combined Terra and Aqua Multi-Angle Implementation of Atmospheric Correction (MAIAC) Land Aerosol Optical Depth level 2 product.
```

File: config/default/common/config/metadata/layers/modis/combined/MODIS_Combined_MAIAC_L2G_AerosolOpticalDepth.md [hunk 2]
```diff
@@ -1 +1,7 @@
+The MAIAC Aerosol Optical Depth layer is from the the Moderate Resolution Imaging Spectroradiometer (MODIS) combined Terra and Aqua Multi-Angle Implementation of Atmospheric Correction (MAIAC) Land Aerosol Optical Depth level 2 product.
```
Comment (edplato) [inline config/default/common/config/metadata/layers/modis/combined/MODIS_Combined_MAIAC_L2G_AerosolOpticalDepth.md:1]:
```suggestion
The MAIAC Aerosol Optical Depth layer is from the Moderate Resolution Imaging Spectroradiometer (MODIS) combined Terra and Aqua Multi-Angle Implementation of Atmospheric Correction (MAIAC) Land Aerosol Optical Depth level 2 product.
```

**Review by edplato — CHANGES_REQUESTED** [review]
👍🏽 Quick changes and looks good to me

**Comment by minniewong** [comment]
@edplato oops. what happens when i cut and paste :)

**Review by edplato — APPROVED** [review]

**Pull request merged**